client/my-sites/stats/stats-upsell/index.tsx

```diff
@@ -109,7 +114,11 @@ export default function StatsUpsell( {
 
 	const onLearnMoreClick = ( event: React.MouseEvent< HTMLButtonElement, MouseEvent > ) => {
 		event.preventDefault();
-		openSupportDoc();
+		if ( isJetpack ) {
+			setTimeout( () => window.open( learnMoreLink, '_blank' ), 250 );
+		} else {
+			openSupportDoc();
+		}
```

Comment:
Is this needed? The "Learn more" button seems specific to WP.com, since it always links to `https://wordpress.com/support/stats/`: https://github.com/Automattic/wp-calypso/blob/81a0dfb07a8c7108dedbf5104bc8970672153814/client/my-sites/stats/stats-upsell/index.tsx

Apparently, Jetpack sites use a different "Learn more" button [here](https://github.com/Automattic/wp-calypso/blob/f5c15f806804757341f88c809121fea278c28c7a/client/my-sites/stats/stats-notices/do-you-love-jetpack-stats-notice.tsx#L165-L179).